numpy/tests/test_reloading.py

```diff
@@ -36,3 +36,30 @@ def test_novalue():
         assert_equal(repr(np._NoValue), '<no value>')
         assert_(pickle.loads(pickle.dumps(np._NoValue,
                                           protocol=proto)) is np._NoValue)
+
+
+def test_py2_can_load_py3_pickle_with_dtype_field_names():
+    # gh-2407 and PR #14275
+    # Py2 should be able to load a pickle that was created in PY3
+    # when the pickle contains a structured dtype with field names
+    import numpy as np
+
+    expected_dtype = np.dtype([('SPOT', np.float64)])
+    expected_data = np.array([(6.0)], dtype=expected_dtype)
+
+    # Pickled under Python 3.6.5 with protocol=2
+    saved_pickle = '''\
```

Comment:
if you prefer, I can also add this as a file to the repo to make it look a bit nicer. Something like `py3_output.pickle`

Reply:
I am fine with the bytestring in the file as-is.